Zopa.ServiceDiagnostics/Health/HealthCheckRunner.cs

```diff
@@ -18,33 +19,38 @@ public HealthCheckRunner(IEnumerable<IAmAHealthCheck> healthChecks, IEnumerable<
 
         public async Task<HealthCheckResults> DoAsync()
         {
+            return await this.DoAsync(CancellationToken.None);
```

Comment:
I'd be more comfortable with one method
`Task<HealthCheckResults> DoAsync(CancellationToken cancellationToken)`, 
or even maybe
`Task<HealthCheckResults> DoAsync(CancellationToken cancellationToken = CancellationToken.None)`
but since we have already mandated changing each check, I don't see the point of not mandating cancellationtokens

Reply:
You can't do 

```csharp
Task<HealthCheckResults> DoAsync(CancellationToken cancellationToken = CancellationToken.None)
```

> Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1736	Default parameter value for 'cancellationToken' must be a compile-time constant	Zopa.ServiceDiagnostics(netcoreapp1.1)	D:\Projects\Zopa.ServiceDiagnostics\Zopa.ServiceDiagnostics\Health\IHealthCheckRunner.cs	8	Active

I'm happy to force a CT by doing 

```csharp
Task<HealthCheckResults> DoAsync(CancellationToken cancellationToken)
```

Reply:
Done